docs/intro/manifesto.md

```diff
@@ -0,0 +1,76 @@
+---
+
+SourceCred is an open-source tool for communities to measure and reward value
+creation.
+
+We're building SourceCred because we believe the way we value things is broken.
+We value people based on their credentials or job titles, not based on the true
+values they bring--especially if that value is "soft value" like emotional
+labor or leadership. We value organizations based on the profits they can
+capture, not the broader effects they have on society. We value the environment
+in terms of the resources we can harvest from it, not on the role it plays in
+sustaining our collective existence.
+
+SourceCred offers a new perspective on value. First, we don't think value is
+universal. It's contextual (intersubjective), and different communities will
+have different perspectives on what is or isn't valuable. For that reason,
+SourceCred is an open-source project which any community can set up, tweak, or
+reconfigure based on their own needs.
+
+Second, we think value isn't about *transactions*, but about *relationships*.
+SourceCred doesn't try to assign a dollar score or price to any contribution or
+person. Instead, every contribution or contributor earns
+[Cred](../concepts/cred.md) based on how it was connected to other
+contributions, and how much Cred those other contributions earned.
+Mathematically, we do this by constructing a network of contributions called a
+Contribution Graph, and then running the CredRank algorithm on it.
+
+Finally, we acknowledge that in order to be sustainable, communities need to
+interact with the "real world" economy of dollars and cents. For that reason,
+every community can issue [Grain](../concepts/grain.md), a community-specific
+cryptocurrency. Grain is distributed based on Cred, so it is all earned by
+community members and contributors. Sponsors purchase Grain, and then use it
+for [Boosting](../concepts/boosting.md), which allows them to influence the
+Cred scores, and thus the community's priorities. Basically, Sponsors help make
+the community economically viable, and in gratitude the community gives
+sponsors some infuence over the community's goals, values, and priorities.
+
+In the short term, SourceCred is focused on supporting open-source crypto
+projects. Crypto projects are an ideal fit because they need a way to
+distribute the tokens that they natively generate, and need a decentralized,
+"new-world" appraoch for doing so--the old world approaches of corporations and
+shareholder value just don't fit. The key early adopter is the SourceCred
+project itself: we have no corporate structure and no investors, but we reward
+everyone in the community via Cred and Grain. (Big thanks to [Protocol
+Labs](https://protocol.ai/) for being our first and biggest sponsor.) We're
+also thrilled to be partnered with other awesome and idealistic projects like
+[MetaGame](https://metagame.wtf/) and
+[AraCred](https://aracred.github.io/website/).
```

Comment:
What about Maker? Building a new financial system is pretty idealistic. I know the partnership isn't official yet, and we don't want to mislead. But it's likely to happen in the next couple weeks, and we may forget to update this. In that vein, I'm generally wary of adding examples to a general intro that may change over time. Easy for this type of thing to get out of date and we're too busy with other things to notice.

Reply:
Yeah, I considered adding Maker as a partner. Didn't want to do it without the consent of the Maker folks. Can try asking about it in our chat with them.

Reply:
Right. Well the Signal poll for the trial ends May 31st (8 days from now) and it looks like it's passing by a landslide. So probably safe to say that will happen. But the word "partner" is tricky (and loaded with baggage from scammy crypto projects announcing "partnerships" based on the slightest interaction with larger companies). Perhaps it's best to just leave out for now, wait on Maker deciding whether to announce via blog post or something.

Reply:
Yeah, agreed.